content/millennium/r4/entities/organization.md

```diff
@@ -150,6 +150,41 @@ The `ETag` response header indicates the current `If-Match` version to use on a
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## $get-cg-for-mrcu
```

Comment:
If I'm reading this right, it looks like we things listed in this order:

- Search
- Retrieve by Id
- Create
- $get-cg-for-mrcu
I think we need to move $get-cg-for-mrcu after Retrieve by Id

Reply:
Updated here: a79abce55312d701d06fe6eec2f9dacedd740bfd